app/src/main/java/com/globant/pokemontcgapp/viewmodel/SplashScreenViewModel.kt

```diff
@@ -0,0 +1,37 @@
+package com.globant.pokemontcgapp.viewmodel
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.globant.pokemontcgapp.contract.SplashScreenContract
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+
+class SplashScreenViewModel : ViewModel(), SplashScreenContract.ViewModel {
+
+    private var mutableMainState: MutableLiveData<SplashScreenData> = MutableLiveData()
```

Comment:
i think this can hold a SplashScreenStatus and avoid th SPlashScreenData

Reply:
changed, thanks